docs/sdk/src/reference_docs/chain_spec_genesis.rs

```diff
@@ -178,7 +178,6 @@
 //! [`pallet::genesis_build`]: frame_support::pallet_macros::genesis_build
 //! [`pallet::genesis_config`]: frame_support::pallet_macros::genesis_config
 //! [`BuildGenesisConfig`]: frame_support::traits::BuildGenesisConfig
-//! [`chain_spec_builder`]: ../../../staging_chain_spec_builder/index.html
```

Comment:
The link is still used in the document a few times. It was moved to the [runtimes](https://github.com/polkadot-fellows/runtimes/tree/main/chain-spec-generator) repo, but not sure if it fits the context here.

Reply:
> The link is still used in the document a few times.

Yes, and the link continues to work because `chain-spec-builder` is [in Cargo.toml](https://github.com/paritytech/polkadot-sdk/blob/c8cb27fbd4920c8369aca053cb92186394d2199d/docs/sdk/Cargo.toml#L37).

> It was moved to the [runtimes](https://github.com/polkadot-fellows/runtimes/tree/main/chain-spec-generator) repo, but not sure if it fits the context here.

I can see `chain-spec-builder` still [inside polkadot-sdk](https://github.com/paritytech/polkadot-sdk/tree/c8cb27fbd4920c8369aca053cb92186394d2199d/substrate/bin/utils/chain-spec-builder) - it's not exactly the same thing as `chain-spec-generator` in `runtimes`, right?

Reply:
Oh yes 🙈  i mixed them up.